frontend/app/templates/create-badges.hbs

```diff
@@ -1,10 +1,13 @@
 <div class="ui stackable grid">
-    <div class="right floated right aligned two wide column">
-        <button {{action 'togglePreview'}} class="tiny ui basic orange button" data-tooltip="Toggle Preview" data-position="left center">
-            <i class="exchange icon"></i>
-        </button>
+    <div class="right floated right aligned two wide column" >
```

Comment:
@abishekvashok  Original code consisted of the static button but now this code consists of the checkbox. :smile:  Please suggest changes required. In case you find it satisfactory Please merge it.